@attr(tags=['advanced'])
def test_16_add_nic_wrong_network_id(self):
    'Add network to vm with wrong network id'
    cmd = addNicToVirtualMachine.addNicToVirtualMachineCmd()
    cmd.virtualmachineid = self.virtual_machine.id
    cmd.networkid = random_gen(id='network_id', size=30)
    with self.assertRaises(Exception) as e:
        self.apiclient.addNicToVirtualMachine(cmd)
        self.debug(('addNicToVirtualMachine API failed with exception: %s' % e.exception))
    return
